language-support/ts/daml-json-types/tsconfig.json

```diff
@@ -2,15 +2,14 @@
   "compilerOptions": {
     "target": "es5",
     "lib": [
-      "dom",
       "es2015"
      ],
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
-    "outDir": "lib/",
+    "outDir": ".",
```

Comment:
error TS18003: No inputs were found in config file '/tmp/tmp.Z8Wk4tg7mz/daml-ledger-fetch/tsconfig.json'. Specified 'include' paths were '["**/*.ts"]' and 'exclude' paths were '["."]'.


Could it be that `eslint` adds `outDir` to `exclude`?

Reply:
I think `tsc` does that. Really annoying. This might impact VS Code support for the TS in our code base negatively.